### DISCUSSION TITLE
Code search in mobile app

### DISCUSSION THREAD

**Discussion by iHarshalAgarwal** [community/community#3429] [Mobile]
If we search for some text, it does not give an option for code search. I think it would be nice to search across code from the mobile app too.

**Comment by PranamBhat** [comment]
Hi @iHarshalAgarwal 

This is possible in Mobile Application. There are few restrictions from **GitHub** as well:

You must be signed into a user account on GitHub to search for code across all public repositories.
Code in forks is only searchable if the fork has more stars than the parent repository. Forks with fewer stars than the parent repository are not indexed for code search. To include forks with more stars than their parent in the search results, you will need to add fork:true or fork:only to your query. For more information, see "Searching in forks."
Only the default branch is indexed for code search.
Only files smaller than 384 KB are searchable.
Only repositories with fewer than 500,000 files are searchable.
Only repositories that have had activity or have been returned in search results in the last year are searchable.
Except with filename searches, you must always include at least one search term when searching source code. For example, searching for language:javascript is not valid, while amazing language:javascript is.
At most, search results can show two fragments from the same file, but there may be more results within the file.
You can't use the following wildcard characters as part of your search query: . , : ; / \ ` ' " = * ! ? # $ & + ^ | ~ < > ( ) { } [ ]. The search will simply ignore these symbols.


For more information, check out here : [https://docs.github.com/en/github/searching-for-information-on-github/searching-code](https://docs.github.com/en/github/searching-for-information-on-github/searching-code)

**Reply by pltrant** [reply]
Yes, please add this feature. I still have to use the the browser from my phone for this functionality.

**Comment by amds9** [comment]
def fibonacci(n)
  return n if ( 0..1 ).include? n
  (fibonacci(n - 1) + fibonacci(n - 2)) #recursive calls
end

**Comment by amds9** [comment]
3429

**Comment by amds9** [comment]
def fibonacci(n)
  return n if ( 0..1 ).include? n
  (fibonacci(n - 1) + fibonacci(n - 2)) #recursive calls
end

**Comment by amds9** [comment]
Authentication

**Comment by amds9** [comment]
- [_********_]()

**Comment by amds9** [comment]
- - [ ] @lynnwallenstein

**Comment by amds9** [comment]
[]()
